### PR TITLE
build: Updated Apache Fileupload to v1.5.0

### PR DESCRIPTION
**Pull request by MMaiero**
Updated dependency and added setFileCountMax(2L)

### CODE REVIEW

File: kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/server/servlet/FileServlet.java [hunk 1]
```diff
@@ -858,6 +858,7 @@ class UploadRequest extends ServletFileUpload {
     public UploadRequest(DiskFileItemFactory diskFileItemFactory) {
         super(diskFileItemFactory);
         setSizeMax(FileServlet.getFileUploadSizeMax());
+        setFileCountMax(2L);
```
Comment (marcellorinaldo) [inline kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/server/servlet/FileServlet.java:861]: Why is this needed? And why are max. 2 files allowed per request?

Comment (marcellorinaldo) [reply kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/server/servlet/FileServlet.java:861]: The first question is answered here: https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-24998

But why 2 files max.? Isn't 1 enough?

**Comment by marcellorinaldo** [comment]
Tested as working on Raspberry Pi 4.

File: kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/server/servlet/FileServlet.java (hunk 1 above)
Comment (MMaiero) [reply kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/server/servlet/FileServlet.java:861]: I have tried with 1 and it does not allow to upload. I have not checked in the code but maybe the limit is exclusive

**Review by marcellorinaldo — APPROVED** [review]

**Pull request merged**